### PR TITLE
Allow bobtimus to use the bitcoind .cookie file for authentication

### LINKED ISSUE #75: Allow to connect to bitcoind via cookie file

### PR DESCRIPTION
**Pull request by thomaseizinger**
Fixes #75.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,9 @@ Feature we will not do:
 
 1. Clone this repo: `git clone https://github.com/coblox/bobtimus.git`
 2. Install dependencies: `yarn`
-3. Copy the default config file to root: `cp ./tests/configs/default.toml ./config.toml`
-4. Modify the appropriate config parameters to ensure Bobtimus can connect to the blockchain nodes
-5. [Make up your own seed words](https://duckduckgo.com/?q=generate+BIP39) and update them in the config file
```
Comment (thomaseizinger) [inline README.md]: We generate seed words automatically now.

**Comment by thomaseizinger** [comment]
> Nice. Is there a similar ticket for btsieve?

I'll open one.

File: src/bitcoin/bitcoinCoreRpc.ts [hunk 2]
```diff
@@ -2,7 +2,12 @@
 import Client from "bitcoin-core";
 import { Transaction } from "bitcoinjs-lib";
 import { getLogger } from "log4js";
-import { BitcoinConfig } from "../config";
+import {
+  BitcoindCoreRpcBasicAuth,
```
Comment (D4nte) [inline src/bitcoin/bitcoinCoreRpc.ts]: 🔧 I find the names a bit confusing are they all seem to be at the same level whereas actually, 2 are for Authentication and one config.

Comment (thomaseizinger) [reply src/bitcoin/bitcoinCoreRpc.ts]: Feel free to propose different names in a follow-up PR :)

**Pull request merged**